package.json

```diff
@@ -1,10 +1,12 @@
 {
   "name": "generator-rn-toolbox",
-  "version": "2.0.6",
```

Comment:
yes, `semantic-release-cli setup` does this by default.
`semantic-release` relies on published version in npm anyway